addons/mail/static/src/components/rtc_call_viewer/rtc_call_viewer.scss

```diff
@@ -8,12 +8,10 @@
     align-items: center;
     justify-content: center;
     flex-direction: column;
-    height: 50%;
     min-height: 50%;
 
     &.o-isMinimized {
-        height: 20%;
```

Comment:
this removal is likely the cause of the bug seen here:

https://youtu.be/o4WAnB6LUSw?t=584